### DISCUSSION TITLE
how to reduce stroke contrast?

### DISCUSSION THREAD

**Discussion by Danyilium** [black7375/Firefox-UI-Fix#768] [Q&A] [answered]
### Is your feature request related to a problem?

![image_2023-09-05_20-53-37](https://github.com/black7375/Firefox-UI-Fix/assets/114657328/f36341f1-bcd7-48a4-ad8f-161151d2a9fd)

### The solution I'd like

how to reduce stroke contrast?

**Comment by black7375** [comment]
What is the theme you are using?
You can check in `Themes` of `about:addons`.

**Reply by Danyilium** [reply]
I use my own theme (created through Firefox Color). has a similar theme here https://addons.mozilla.org/uk/firefox/addon/breezedarklighttheme/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search

**Reply by Danyilium** [reply]
just wanted to make a theme like Firefox quantum was by default
![зображення](https://github.com/black7375/Firefox-UI-Fix/assets/114657328/85488191-4a06-41b2-9c9b-4d4405cbfe3a)

**Reply by black7375** [reply, marked as answer]
You can set with `Toolbar Bottom Separator`
![image](https://github.com/black7375/Firefox-UI-Fix/assets/25581533/d1830387-d590-4d30-8d14-b320fcdc2e80)

---

How about using photon colors?
- https://addons.mozilla.org/en-US/firefox/addon/photon-colors/

**Reply by Danyilium** [reply]
Thank you!
